frontend/src/pages/RunDetailsV2.tsx

```diff
@@ -294,14 +294,23 @@ function updateToolBarActions(
 }
 
 function getDetailsFields(apiRun?: ApiRun): Array<KeyValue<string>> {
+  // check if the run has finished or not. The default value for apiRun.finished_at is
+  // Date(0), when it is not specified.
+  var finished_at = new Date(0);
+  if (apiRun?.finished_at) {
+    finished_at = new Date(apiRun?.finished_at);
+  }
+  var time_zero = new Date(0);
+  var has_finished = Boolean(finished_at.getTime() !== time_zero.getTime());
```

Comment:
I think the "has_finished" is not necessary (but good to have) for this case. You can also directly put this conditional operator into line 313 & 314.